### PR TITLE
Added http-server & http-server documentation

### LINKED ISSUE #1018: Super simple lightweight http server to share files/pages -- http-server
I use this container to host a file server (locally) to access my shared files, especially to sideload APKs to my FireTV boxes and the like.  Might be very limited in functionality and usefulness, but I figured I'd share my overrides for people to review.

these are not really needed, but I made my override identical to the default apps:
     - ${DOCKERSHAREDDIR}:/shared
     - ${DOCKERCONFDIR}/http-server:/config:rw

this might want to be a different location, but this is where I put mine:
     - ${DOWNLOADSDIR}/http-server:/html:rw
Annotation 2020-07-29 091844

-------------------------------------------------------

  http-server:
    environment:
     - PGID=${PGID}
     - PUID=${PUID}
     - TZ=${TZ}
     - FILE_SERVER_TITLE=${DOCKERHOSTNAME} HTTP File Server
    hostname: ${DOCKERHOSTNAME}
    image: patrickdappollonio/docker-http-server
    ports:
     - 5000:5000
    restart: unless-stopped
    logging:
      driver: json-file
      options:
        max-file: ${DOCKERLOGGING_MAXFILE}
        max-size: ${DOCKERLOGGING_MAXSIZE}
    volumes:
     - /etc/localtime:/etc/localtime:ro
     - ${DOCKERSHAREDDIR}:/shared
     - ${DOCKERCONFDIR}/http-server:/config:rw
     - ${DOWNLOADSDIR}/http-server:/html:rw
    container_name: http-server

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1018 

**Approach**
How does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [ ] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/httpserver/httpserver.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+services:
+  httpserver:
+    container_name: httpserver
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - FILE_SERVER_TITLE=${HTTPSERVER_FILE_SERVER_TITLE}
+      - FILE_SERVER_COLOR_SET=${HTTPSERVER_FILE_SERVER_COLOR_SET}
```
Comment (nemchik) [inline compose/.apps/httpserver/httpserver.yml:9]: Sort these.

File: compose/.apps/httpserver/httpserver.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    logging:
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERSHAREDDIR}:/html/shared
```
Comment (nemchik) [inline compose/.apps/httpserver/httpserver.yml:18]: The shared volume should be the same on all containers in the project. This doesn't match. We should use a different variable for this folder if it's needed.

Comment (angrycuban13) [reply compose/.apps/httpserver/httpserver.yml]: Fixed on 222faff

File: compose/.apps/httpserver/httpserver.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/httpserver/httpserver.yml]: Fixed on 222faff

File: compose/.apps/httpserver/httpserver.labels.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+services:
+  httpserver:
+    labels:
+      com.dockstarter.appinfo.description: "http-server is a simple binary to provide a static http server from a given folder."
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.nicename: "httpserver"
+      com.dockstarter.appvars.http_html_dir: ""
```
Comment (nemchik) [inline compose/.apps/httpserver/httpserver.labels.yml:7]:
```suggestion
      com.dockstarter.appvars.httpserver_html_dir: "/mnt/httpserver"
```

Comment (nemchik) [reply compose/.apps/httpserver/httpserver.labels.yml:7]: Also sort alphabetically after this change.

File: compose/.apps/httpserver/httpserver.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    logging:
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERSHAREDDIR}:/shared
+      - ${HTTP_HTML_DIR}/html
```
Comment (nemchik) [inline compose/.apps/httpserver/httpserver.yml:19]:
```suggestion
      - ${HTTPSERVER_HTML_DIR}/html

```

Comment (angrycuban13) [reply compose/.apps/httpserver/httpserver.yml]: Fixed on 78966d0

I also forgot the `:` and caught it before you 😝

File: compose/.apps/httpserver/httpserver.labels.yml (hunk 3 above)
Comment (angrycuban13) [reply compose/.apps/httpserver/httpserver.labels.yml]: Fixed on 78966d0

**Pull request merged**